### PR TITLE
Cleanup consecutive entries code from window_service

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
WindowService and DbWindow have a lot of unused code from when WindowService used to forward consecutive entries to DbLedger.

#### Summary of Changes
Remove entries being returned on insertion from DbLedger, ReplayStage now queries the ledger for entries.

Remove the logic in DbWindow and WindowService surrounding managing and returning consecutive entries.

Remove the "max_height" logic from WindowService as it no longer tracks consecutive entries in the DbLedger. @sakridge 

Fixes #

### CODE REVIEW

File: src/replicator.rs [hunk 1]
```diff
@@ -156,7 +155,7 @@ impl Replicator {
             Self::poll_for_last_id_and_entry_height(&cluster_info)?;
 
         let signature = keypair.sign(storage_last_id.as_ref());
-        let (entry_height, max_entry_height) =
+        let (entry_height, _max_entry_height) =
             get_entry_heights_from_last_id(&signature, storage_entry_height);
```
Comment (garious) [inline src/replicator.rs]: Looks like this is the only use of this function. Why not change it to return just the one value?

File: src/db_window.rs [hunk 2]
```diff
@@ -214,10 +212,6 @@ pub fn process_blob(
     leader_scheduler: &Arc<RwLock<LeaderScheduler>>,
     blocktree: &Arc<Blocktree>,
     blob: &SharedBlob,
-    max_ix: u64,
```
Comment (sakridge) [inline src/db_window.rs]: Can you fix the comment on this function?

File: src/db_window.rs [hunk 3]
```diff
@@ -239,54 +233,24 @@ pub fn process_blob(
     }
 
     // Insert the new blob into the window
-    let mut consumed_entries = if is_coding {
+    if is_coding {
```
Comment (sakridge) [inline src/db_window.rs]: Fix comment.

File: src/blocktree.rs [hunk 4]
```diff
@@ -476,21 +476,19 @@ impl Blocktree {
...
+        self.insert_data_blobs(blobs)
     }
 
-    pub fn write_blobs<I>(&self, blobs: I) -> Result<Vec<Entry>>
+    pub fn write_blobs<I>(&self, blobs: I) -> Result<()>
     where
         I: IntoIterator,
         I::Item: Borrow<Blob>,
     {
         //let blobs = blobs.into_iter().map(|b| *b.borrow());
```
Comment (sakridge) [inline src/blocktree.rs]: Remove?

File: src/db_window.rs [hunk 5]
```diff
@@ -214,10 +212,6 @@ pub fn process_blob(
     leader_scheduler: &Arc<RwLock<LeaderScheduler>>,
```
Comment (sakridge) [inline src/db_window.rs]: Delete or comment on out-dated generate_repairs fn please

File: src/replicator.rs (hunk 1 above)
Comment (carllin) [reply src/replicator.rs]: Wasn't sure if that max_entry_height would still have some utility in the future. As Steven has acknowledged it doesn't, I'll remove it now :)

Comment (garious) [reply src/replicator.rs]: Delete anything and everything that's not in use. It's all in git. We know where to find it if we want it back.

File: src/db_window.rs (hunk 5 above)
Comment (sakridge) [reply src/db_window.rs]: @carllin generate_repairs is still there, no?

Comment (sakridge) [reply src/db_window.rs]: also calculate_max_repair_entry_height doesn't seem to be used either.

**Pull request merged**